content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Lambda Function Log Collection Troubleshooting Guide
+kind: documentation
+---
+
+## Logs from lambda functions are not visible in Log Explorer page
```

Comment:
```suggestion
## Logs from Lambda functions are not visible in Log Explorer page
```

Reply:
I would actually remove this title as it is kind of a duplicate of the main article title. WDYT?

Reply:
What about getting rid of the title and changing the line after it to:

>If you don't see logs forwarded from a lambda function in the Log Explorer, see the Datadog-AWS Log integration to configure your environment.

Reply:
Sounds like a good plan indeed